bazel/sh_test_wrapper.sh

```diff
@@ -3,4 +3,6 @@
 # Where the runfiles are for tests.
 export TEST_RUNDIR="${TEST_SRCDIR}/${TEST_WORKSPACE}"
 
+cd $(dirname "$0")
```

Comment:
I tried one other way, which is to use the fact that a test with `tags = ["manual"]` will not be included when running tests with wildcards (e.g. `bazel test //test/...`).

My idea was to pass a `select()` to tags and add the manual tag when hot restart is disabled. But that's not supported either:

```
/Users/stephan/workspace/turbinelabs-envoy/test/integration/BUILD:14:1: //test/integration:hotrestart_test: attribute "tags" is not configurable.
```